doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -313,6 +327,18 @@ Additional scenario XML elements:
   PCI devices list of the VM; it is hard-coded for each scenario so it
   is not configurable for now.
 
+``mmio_resources``:
+  MMIO resources to passthrough.
+
+``TPM2`` (a child node of ``mmio_resources``):
+  TPM2 device to passthrough.
+
+``p2sb`` (a child node of ``mmio_resources``):
+  P2SB device to passthrough.
+
+``pt_intx``:
+  pt intx mapping.
```

Comment:
This is mostly a repeat of the name of the parameter. Can you describe this a little more in plain english, i.e. Passthrough Interrupt mapping.

Reply:
I agree that these descriptions could be explained better, even if it's just expanding the acronyms.

Reply:
I will add more information